src/js/languages/default.json

```diff
@@ -66,6 +67,9 @@
     "pauseButton": "pause",
     "playButton": "play",
     "volumeDown": "volume down",
-    "volumeUp": "volume up"
+    "volumeUp": "volume up",
+    "description": "video audio description",
```

Comment:
Since we are referring to it as "Video description" below, should we just say "video description" here? OR you mentioned that "Audio description" is more common, should we be saying that across the board?

Reply:
Leaning towards "Audio description" myself after doing some quick googling.